app/lib/efile/tax_calculator.rb

```diff
@@ -4,22 +4,19 @@ class TaxCalculator
 
     private
 
-    def set_line(line_id, value_fn)
+    def set_line(line_id, value_fn_or_data_source, field = nil)
       source_description = nil
-      if value_fn.is_a?(Symbol)
-        value_fn = method(value_fn)
+      if field
```

Comment:
Since `field` is only available when `value_fn` is not a callable, I wonder if there's a way to avoid the optional argument. Not a blocker.